src/libs/notifications.js

```diff
@@ -74,134 +73,130 @@ const NotificationDisplay = ({ id }) => {
-                display: 'flex',
-                flexDirection: 'column',
-                fontSize: 12,
-              },
-              role: 'alert',
-              'aria-labelledby': labelId,
-              'aria-describedby': message ? descId : undefined,
-            },
-            [
-              // content and close button
-              div({ style: { display: 'flex', padding: '0.75rem 1rem' } }, [
-                // content
-                div({ style: { display: 'flex', flex: 1, flexDirection: 'column', overflow: 'hidden' } }, [
-                  // icon and title
-                  div({ style: { display: 'flex' } }, [
-                    !!iconType &&
-                      icon(iconType, {
-                        'aria-hidden': false,
-                        'aria-label': ariaLabel,
-                        size: 26,
-                        style: { color: baseColor(), flexShrink: 0, marginRight: '0.5rem' },
-                      }),
-                    div({ id: labelId, style: { fontWeight: 600, overflow: 'hidden', overflowWrap: 'break-word' } }, [title]),
-                  ]),
-                  !!message && div({ id: descId, style: { marginTop: '0.5rem', overflowWrap: 'break-word' } }, [message]),
-                  div({ style: { display: 'flex' } }, [
-                    !!detail &&
-                      h(
-                        Clickable,
-                        {
-                          style: { marginTop: '0.25rem', marginRight: '0.5rem', textDecoration: 'underline' },
-                          onClick: () => setModal(true),
-                        },
-                        ['Details']
-                      ),
-                    !!action &&
-                      h(
-                        Clickable,
-                        {
-                          style: { marginTop: '0.25rem', textDecoration: 'underline' },
-                          onClick: () => {
-                            action.callback();
-                            Store.removeNotification(id);
-                          },
-                        },
-                        [action.label]
-                      ),
-                  ]),
-                ]),
-                h(
-                  Link,
-                  {
-                    style: { alignSelf: 'start' },
-                    'aria-label': type ? `Dismiss ${type} notification` : 'Dismiss notification',
-                    title: 'Dismiss notification',
-                    onClick: () => {
-                      Store.removeNotification(id);
-                      onDismiss?.();
-                    },
-                  },
-                  [icon('times', { size: 20 })]
-                ),
-              ]),
-              notifications.length > 1 &&
-                div(
-                  {
-                    style: { alignItems: 'center', borderTop: `1px solid ${baseColor()}`, display: 'flex', fontSize: 10, padding: '0.75rem 1rem' },
+  return div(
```

Comment:
This looks like a large change in Github, but I removed the two outer react-hyperscript-helper functions and replaced their usage of IdContainer with two uses of useUniqueId.

Reply:
Reviewing with whitespace changes ignored makes the diff much smaller.
https://github.com/DataBiosphere/terra-ui/pull/4609/files?w=1

Reply:
Oooo, I'll have to try to remember that one. Thank you.